core/src/main/java/org/stellar/anchor/apiclient/PlatformApiClient.java

```diff
@@ -49,17 +41,33 @@ public PlatformApiClient(AuthHelper authHelper, String endpoint) {
    * @throws IOException if the request fails due to IO errors.
    * @throws AnchorException if the response is not successful.
    */
+  @Deprecated // ANCHOR-641 Use getTransactionByRpc instead
   public GetTransactionResponse getTransaction(String id) throws IOException, AnchorException {
     Request request = getRequestBuilder().url(endpoint + "/transactions/" + id).get().build();
     String responseBody = handleResponse(client.newCall(request).execute());
     return gson.fromJson(responseBody, GetTransactionResponse.class);
   }
 
+  public GetTransactionResponse getTransactionByRpc(String id) throws IOException, AnchorException {
```

Comment:
Is there a use case that we have to keep the deprecated API in the `PlatformApiClient` class or interface? 

I think if the `PlatformApiClient` do not have to support the deprecated endpoint. We may change the `getTransaction()` implementation as RPC. 

Consider the pattern where the `PlatformClientApi` is an interface, it would be better to replace the implementation instead of adding additional functions.

Reply:
I will complete this ticket in separate PRs as rpc methods require some different logic like error handling. 

This is the 1st one that implement new endpoint. 
Next step is migrate existing usage of old REST endpoints to new one and fix tests.
The last step is to replace/rename the rpc endpoint and get rid of the old one.

Let me know if this PR looks good to you and I can move to next step